cmd/root.go

```diff
@@ -54,4 +93,42 @@ func Execute() {
 	}
 
 	Cron(cfg)
+
+	if err = ctxPool.Wait(); err != nil {
+		log.Panic().Err(err).Msg("error in goroutine pool")
```

Comment:
To be consistent with the other logs:
```suggestion
		log.Panic().Err(err).Msg("Error in goroutine pool")
```